extension/source/background/PreferencesController.ts

```diff
@@ -89,14 +116,14 @@ export default class PreferencesController {
     const $preferences = await this.preferences.read();
```

Comment:
#nitpick Not from this changeset, but would prefer a different const name than one that starts with `$`.